compat/mingw.c

```diff
@@ -2208,179 +2208,18 @@ int mingw_putenv(const char *namevalue)
 
 #endif
 
-/*
```

Comment:
The changes in this commit look all good, but maybe it would make sense to stress more the *why* than the *what* in the commit message. For example, the commit subject could say `mingw: remove obsolete IPv6-related code`. And the body of the commit message could talk about the previous commit removing the users of this code.